versions/1.0.0.md

```diff
@@ -168,14 +170,14 @@ updates:
     description: This is the updated description
 - target: paths."/bar"
   merge:
-      post:
-          description: This is an updated description of a child object
-          x-safe: false
+    post:
+      description: This is an updated description of a child object
+      x-safe: false
 ```
 
 #### Wildcard Overlays Examples
 
-One significant advantage of using the JMESPath syntax that it allows referencing multiple nodes in the target document.  This would allow a single update object to be applied to multiple target objects using wildcards.
+One significant advantage of using the JMESPath syntax is that it allows referencing multiple nodes in the target document.  This would allows a single update object to be applied to multiple target objects using wildcards.
```

Comment:
`This allows` or `This would allow`?

Reply:
```suggestion
One significant advantage of using the JMESPath syntax is that it allows referencing multiple nodes in the target document.  This would allow a single update object to be applied to multiple target objects using wildcards.
```